desk/app/chat.hoon

```diff
@@ -156,7 +156,7 @@
     |=  =chat:zero
     %*  .  *chat:one
       remark  remark.chat
-      log     log.chat
+      ::  log     *log.chat
       perm    perm.chat
       pact    pact.chat
```

Comment:
I just ripped out all the state migrations in jake's branch since we're nuking anyway